torch/testing/_internal/common_methods_invocations.py

```diff
@@ -20954,6 +20955,14 @@ def __init__(
         ),
     ),
     #
+    # torch.linalg
+    #
+    ReductionPythonRefInfo(
```

Comment:
Can we still add the scalar tensor case described above to the sample inputs for vector norm?

Reply:
Is this not already tested in the regular reduction ops? If it is, it should be, shouldn't it?